### PR TITLE
bug fix for migration from azure blob failed.

### PR DESCRIPTION
**Pull request by sfzeng**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: s3/pkg/meta/db/drivers/tidb/bucket.go [hunk 1]
```diff
@@ -453,12 +453,12 @@ func (t *TidbClient) CountObjects(ctx context.Context, bucketName, prefix string
 	var err error
 	var sizeStr sql.NullString
 	if prefix == "" {
-		sqltext = "select count(*),sum(size) from objects where bucketname=?;"
-		err = t.Client.QueryRow(sqltext, bucketName).Scan(&rsp.Count, &sizeStr)
+		sqltext = "select count(*),sum(size) from objects where bucketname=? and tier<?;"
+		err = t.Client.QueryRow(sqltext, bucketName, utils.Tier999).Scan(&rsp.Count, &sizeStr)
```
Comment (kumarashit) [inline s3/pkg/meta/db/drivers/tidb/bucket.go]: Why only Tier999?

Comment (sfzeng) [reply s3/pkg/meta/db/drivers/tidb/bucket.go]: That's tier for archived.

Comment (sfzeng) [reply s3/pkg/meta/db/drivers/tidb/bucket.go]: Archived data cannot be copied.

Comment (kumarashit) [reply s3/pkg/meta/db/drivers/tidb/bucket.go]: Ok

**Pull request merged**